### PR TITLE
Seeding plant data

### PR DESCRIPTION
**Pull request by bentrzcinski**

### CODE REVIEW

**Review by Fahi28 — APPROVED** [review]

File: schema.sql [hunk 1]
```diff
@@ -306,4 +306,57 @@ VALUES
...
+    (34, 2, 33, 163),
+    (35, 1, 34, 35),
+    (36, 2, 35, 185),
+    (37, 3, 36, 77),
+    (38, 2, 37, 171),
+    (39, 1, 38, 85),
+    (40, 3, 39, 61),
+    (41, 1, 40, 77),
+    (42, 1, 41, 26),
+    (44, 2, 42, 112),
```
Comment (jiuliangut) [inline schema.sql:354]: missing plant_id 43 - is this expected?

**Review by jiuliangut — APPROVED** [review]
beautifully planted seeds

File: schema.sql (hunk 1 above)
Comment (bentrzcinski) [reply schema.sql:354]: good spot! Yeah apparently its on loan to another museum (according to the endpoint)

**Review by anitamegarry — APPROVED** [review]
Wow great work!

**Pull request merged**